compiler/testData/ir/irText/expressions/callableReferences/suspendConversion.kt

```diff
@@ -1,6 +1,7 @@
 // !LANGUAGE: +SuspendConversion
 
 fun useSuspend(fn: suspend () -> Unit) {}
+fun useSuspendNullable(fn: (suspend () -> Unit)?) {}
```

Comment:
Can you also add a nested case like `fun useSuspendNullable(fn: ((suspend () -> Unit)?)?) {}`?

Reply:
Just curious, how can I read that nested nullable type for `fn`? Why aren't they any differences in generated IR texts and rendered types?

Reply:
There's no difference because the type is both `kotlin.coroutines.SuspendFunction0<kotlin.Unit>?` regardless of the extra nullable type. `T?? === T?`

The extra `()?` nested is superfluous and in fact the IDE tells you it's redundant (not sure if it's IDE-only intention or coming from the compiler).